tests/unit/cli/test_review_command.py

```
@@ -262,6 +263,214 @@ def test_review_exits_zero_without_p1_findings() -> None:
+                name="openai",
+            ),
+        ),
+        "run_review": patch(
+            "lintro.cli_utils.commands.review.run_review",
+            return_value=_empty_result(),
+        ),
+        "render_review_output": patch(
+            "lintro.cli_utils.commands.review.render_review_output",
+        ),
+    }
+    return patches
```

Ruff (0.15.20)

[warning] Unnecessary assignment to `patches` before `return` statement

Remove unnecessary assignment

(RET504)